codec-http2/src/main/java/io/netty/handler/codec/http2/DelegatingDecompressorFrameListener.java

```diff
@@ -239,8 +241,8 @@ private void initDecompressor(int streamId, Http2Headers headers, boolean endOfS
         }
     }
 
-    private static Http2Decompressor decompressor(Http2Stream stream) {
-        return (Http2Decompressor) (stream == null? null : stream.getProperty(Http2Decompressor.class));
+    private Http2Decompressor decompressor(Http2Stream stream) {
```

Comment:
This is used in inner class (not that you introduced this).  Make package protected.